debian/gbp.conf

```diff
@@ -0,0 +1,5 @@
+[DEFAULT]
+debian-tag=v%(version)s
+
+[buildpackage]
+export=INDEX
```

Comment:
Why you need this?

Reply:
The idea is to be able to build a source package with staged content, like you git add some files, didn’t commit and `gpb buildpackage -S`, so that you can test from a build.